notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1396 @@
+{
```

Comment:
<ul><li>Since the part of the notebook relative to deploying the model is just some necessary setup for model monitory, what about packaging the deployment code in a python script using the preferred Python SDK and just invoking that script at the very beginning of the notebook? </li></ul><p> </p><ul><li>Same comment as above relative to <code>beta</code></li><li>I'd add blank lines between the command for readability </li><li>Is the <code>traffic-split</code> argument necessary? If not, remove it, otherwise it deserve some explanation </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>